ui.tests/test-module/specs/dropdown/dropdown.authoring.cy.js

```diff
@@ -149,6 +149,19 @@ describe('Page - Authoring', function () {
             getPreviewIframeBody().find('.cmp-adaptiveform-dropdown').parent().contains('Item 1').should('not.exist');
             cy.deleteComponentByPath(dropdown);
         });
+
+        it('enable suggestions of dropdown component', function () {
+            insertDropDownInContainer();
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + dropDownEditPathSelector);
+            cy.invokeEditableAction("[data-action='CONFIGURE']");
+            cy.get('input[name="./fd:enableSuggestions"]').should('exist').check();
+            cy.get('.cq-dialog-submit').click();
+
+            getPreviewIframeBody().find('input[list]').should('exist');
```

Comment:
Can we also add test cases for other use-cases with suggestions enabled, like placeholder , default selection etc

Reply:
sure, will add test cases for to cover all these cases.